### PR TITLE
Delete Clone Support

### PR DESCRIPTION
**Pull request by drekle**
Currently users that create a clone that fails cannot delete that clone through the API.

### CODE REVIEW

**Review by dclear — APPROVED** [review]

File: live/sites/v1alpha1/site.proto [hunk 1]
```diff
@@ -234,6 +234,17 @@ message DescribeCloneResponse {
...
 
+message DeleteCloneRequest {
+  /*
+  `Required`
+  Reference to the clone site operation 
+  */
+  string operationRef = 1;
+}
+
+message DeleteCloneResponse {
```
Comment (drekle) [inline live/sites/v1alpha1/site.proto:245]: Don't know what I would return here, potentially the name but this is just going to be copied from the request anyway.  Left blank to expand upon in the future to just get the support out for now.

Comment (wozniakjan) [reply live/sites/v1alpha1/site.proto:245]: I think empty response is fine.

**Review by wozniakjan — APPROVED** [review]
lgtm

**Review by nic-hima — APPROVED** [review]
lgtm. interesting how the doc pages got reshuffled/organized. Only addition seems like 8 lines to `doc/api/site-api.md` for the DeleteClone Request/Response.

**Comment by drekle** [comment]
> lgtm. interesting how the doc pages got reshuffled/organized. Only addition seems like 8 lines to `doc/api/site-api.md` for the DeleteClone Request/Response.

Yeah, its fairly annoying for me as I have changes in administration docs that I did not touch.  There might be an option to the renderer.

**Pull request merged**